spring/src/main/java/io/cloudevents/spring/http/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Provides classes related to working with Cloud Events within the context of Spring MVC.
```

Comment:
In this package we handle http generic right? Not only MVC?

Reply:
It's shared between MVC and Webflux, so yes, its generic.